exchange/exchange.go

```diff
@@ -293,6 +301,12 @@ func (e *exchange) HoldAuction(ctx context.Context, r AuctionRequest, debugLog *
 	var bidResponseExt *openrtb_ext.ExtBidResponse
 	if anyBidsReturned {
 
+		//If floor enforcement config enabled then filter bids
+		adapterBids, bidRejections := enforceFloors(&r, adapterBids, e.floor, conversions, responseDebugAllow)
```

Comment:
If it's easier to return a slice of errors instead of bidRejections, feel free to go for it.

Reply:
returned slice of errors